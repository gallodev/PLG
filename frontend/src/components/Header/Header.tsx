import React from 'react'
import { ContainerLogo, TopHeader, Title } from './Styled';
import LogoRD from '../../assets/logo-rd-station.png';
import LogoSpaceX from "../../assets/logo-spacex.png";

const Header: React.FC = () => {
    return (
        <TopHeader>
            <ContainerLogo>
                <ul>
                    <li><img src={LogoSpaceX} width="120" height="40" /></li>
                    <li><img src={LogoRD} width="120" height="40" /></li>
                </ul>
            </ContainerLogo>
            <Title> ðŸš€ Space X launches</Title>
        </TopHeader>
    )
}

export { Header };