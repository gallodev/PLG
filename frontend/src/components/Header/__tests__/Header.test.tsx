import { render } from '@testing-library/react';
import { Header } from '../Header';

describe('[Header] Should test Header component', () => {
 it('[Header] Should render Header and match snapshot', () => {
     const { asFragment } = render(<Header />);  
     expect(asFragment()).toMatchSnapshot();
 });
});
