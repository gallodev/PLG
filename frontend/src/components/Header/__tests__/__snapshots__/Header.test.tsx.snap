// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Header] Should test Header component [Header] Should render Header and match snapshot 1`] = `
<DocumentFragment>
  <header
    class="sc-gsTEea kjDHvi"
  >
    <div
      class="sc-bdfBQB hhVVxk"
    >
      <ul>
        <li>
          <img
            height="40"
            src="logo-spacex.png"
            width="120"
          />
        </li>
        <li>
          <img
            height="40"
            src="logo-rd-station.png"
            width="120"
          />
        </li>
      </ul>
    </div>
    <h1
      class="sc-dlfnuX hbaWER"
    >
      ðŸš€
    </h1>
  </header>
</DocumentFragment>
`;
